SRC/arpack.pc.in

```diff
@@ -9,4 +9,4 @@ Version: @PACKAGE_VERSION@
 URL: @PACKAGE_URL@
 Libs: -L${libdir} -larpack@LIBSUFFIX@
 Libs.private: @ARPACK_PC_LIBS_PRIVATE@
-Cflags: -I${includedir}/arpack${SUFFIX64}
+Cflags: -I${includedir}/arpack@SUFFIX64@
```

Comment:
You'll need to move the definition of `SUFFIX64` in `CMakeLists.txt` up a few lines. Or it isn't defined yet when the .pc files are created.

Reply:
Thank you and thank you for your patience